python_modules/dagster-graphql/dagster_graphql/schema/backfill.py

```diff
@@ -325,6 +353,11 @@ class Meta:
         graphene.NonNull("dagster_graphql.schema.instigation.GrapheneInstigationEventConnection"),
         cursor=graphene.String(),
     )
+    jobName = graphene.String()  # for RunsFeedEntry interface - dupe of partitionSetName
+    assetCheckSelection = graphene.List(
+        graphene.NonNull("dagster_graphql.schema.asset_checks.GrapheneAssetCheckHandle")
+    )
+    runType = graphene.NonNull(GrapheneRunsFeedEntryType)
```

Comment:
i dont think this enum provides any utility, we will have access to what the concrete types are